test/k8sT/Nightly.go

```diff
@@ -34,7 +34,8 @@ import (
 )
 
 var (
-	configMap = "ConfigMap"
+	configMap       = "ConfigMap"
+	endpointTimeout = (60 * time.Second)
```

Comment:
The timeout is changed from 300 seconds to 60 seconds - why this big of a change in the timeout value?

Reply:
Mainly, sometimes the endpoints fails due timeout but we want to measure the data. So having a small timeout get the test in the fail status. I want to have all that data in our graphs and measure it over time.